### ISSUE TITLE
Serverless already accepts external sns events

### ISSUE THREAD

**Issue by alejobs** [silvermine/serverless-plugin-external-sns-events#12]
Please take a look at: https://github.com/serverless/serverless/pull/2796

So I think now this plugin is useless.

**Comment by jthomerson** [comment]
Wow @alejobs ... not cool. How is saying "this plugin is useless" helpful? What's the intent of opening the issue?

You're correct that SLS recently *merged* some support for this, but:

 1. It hasn't actually been released yet.
 2. Even if it was released, many people are running on an older version of SLS and will still need this until they can upgrade.
 3. Even when someone upgrades to the newer / next version of SLS, they may not necessarily have time to change to the new format and use that functionality (although they should try to do so ASAP).

This plugin will eventually go away and stop being supported - after some time has passed to allow for folks to upgrade and depend on the SLS functionality. If you'll take note of the other two plugins that we've released for Serverless, we specifically mentioned in the readme file that they would go away after "native" support was added.
